db/models/Place.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+// placeholders for now
+const categories = ['restaurant, cafe']
+const accesibilityOptions = ['braille menu']
+
+const placeSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: String,
+    description: String,
+    address: String,
+    lat: Number,
```

Comment:
Yeah, fairly confusing.

https://stackoverflow.com/questions/32199658/create-find-geolocation-in-mongoose

https://github.com/Automattic/mongoose/tree/master/examples/geospatial

Are they helpful?